recipe/meta.yaml

```diff
@@ -41,8 +41,8 @@ requirements:
     - lz4-c
     - prometheus-cpp
     - double-conversion
-    - libevent {{ libevent }}
-    - libprotobuf <4
+    - libevent
+    - libprotobuf
```

Comment:
Due to this cap, #57 has zero effect

Reply:
libprotobuf < 4 had to be used for consistency with other constraints than the ones of conda-forge. I'll see if I can relax it now.